fields/types/location/LocationType.js

```diff
@@ -368,6 +368,10 @@ function doGoogleGeocodeRequest(address, region, callback) {
 		options.region = region;
 	}
 
+	if (keystone.get('google server api key')){
```

Comment:
This addition broke location auto-detect on my site - pressing 'Autodetect and improve location on save' would no longer geolocate the address. Tracked it down to this commit in version 0.3.13 and up.
If I delete the above lines of code it works again.

Could this be removed/tweaked?